biotrainer/config/embedding_options.py

```diff
@@ -144,5 +145,60 @@ def allow_download(self) -> bool:
         return True
 
 
+class DimensionReduction(EmbeddingOption, ConfigOption):
+
+    @classproperty
+    def name(self) -> str:
+        return "dimension_reduction_method"
+
+    @property
+    def default_value(self) -> Union[str, int, float, bool, Any]:
+        return ""
+
+    @classproperty
+    def allow_multiple_values(self) -> bool:
+        return False
+
+    @property
+    def possible_values(self) -> List[Any]:
+        return ["umap", "tsne"]
+
+    @classproperty
+    def allowed_protocols(self) -> List[Protocol]:
+        return [Protocol.sequence_to_class, Protocol.sequence_to_value]
+
+    @classproperty
+    def required(self) -> bool:
+        return False
+
+
+class ReducedDimension(EmbeddingOption, ConfigOption):
+
+    @classproperty
+    def name(self) -> str:
+        return "reduced_dimension"
```

Comment:
I am thinking about whether "reduced_dimensions" or even "reduce_to_dimensions" would be indicating a bit clearer what is done by this config option, what do you think?

Reply:
how about n_reduced_components?